sonarts-sq-plugin/sonar-typescript-plugin/src/main/java/org/sonar/plugin/typescript/rules/NoImplicitDependencies.java

```diff
@@ -19,19 +19,31 @@
  */
 package org.sonar.plugin.typescript.rules;
 
+import com.google.gson.JsonArray;
 import com.google.gson.JsonElement;
+import java.util.Arrays;
 import org.sonar.check.Rule;
+import org.sonar.check.RuleProperty;
 
 @Rule(key = "S4328")
 public class NoImplicitDependencies extends TypeScriptRule {
 
+  private static final String DEFAULT_WHITELIST = "";
+
+  @RuleProperty(key = "whitelist",
+    description = " Comma separated list of modules to skip checking their existence in package.json.",
```

Comment:
there is an extra space at the beginning. Intended?

Wording sounds strange. What about `Comma separated list of modules to ignore while checking in package.json`.

Reply:
Not intended indeed and I agree the wording is not clear, I took it from TSLint doc, but yours looks more clear. Thanks!